meta/heart-bits/contao-columns-bundle/en.yml

```
@@ -0,0 +1,10 @@
+en:
+    title: Front end-Grid
```

```suggestion
    title: Front end grid
```
Or `Front End Grid`. But you usually do not use `-` as in German.